scripts/ci/check_compliance.py

```diff
@@ -1266,12 +1266,18 @@ def check_no_undef_outside_kconfig(self, kconf):
         "FOO_SETTING_1",
         "FOO_SETTING_2",
         "GEN_UICR_GENERATE_PERIPHCONF", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_PROTECTEDMEM", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_PROTECTEDMEM_SIZE_BYTES", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_SECONDARY", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_SECONDARY_GENERATE_PERIPHCONF", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_SECONDARY_PROCESSOR_VALUE", # Used in specialized build tool, not part of main Kconfig
-        "GEN_UICR_SECURESTORAGE", # Used in specialized build tool, not part of main Kconfig
+        "GEN_UICR_PROTECTEDMEM",
+        "GEN_UICR_PROTECTEDMEM_SIZE_BYTES",
+        "GEN_UICR_SECONDARY",
+        "GEN_UICR_SECONDARY_GENERATE_PERIPHCONF",
+        "GEN_UICR_SECONDARY_PROCESSOR_VALUE",
+        "GEN_UICR_SECONDARY_WDTSTART",
+        "GEN_UICR_SECONDARY_WDTSTART_CRV",
+        "GEN_UICR_SECONDARY_WDTSTART_INSTANCE_CODE",
+        "GEN_UICR_SECURESTORAGE",
+        "GEN_UICR_WDTSTART",
+        "GEN_UICR_WDTSTART_CRV",
+        "GEN_UICR_WDTSTART_INSTANCE_CODE", # Used in specialized build tool, not part of main Kconfig
```

Comment:
nit: is the purpose here to only have the comment on the last entry? If so the top entry still has the comment.

Reply:
The problem with having at the beginning and end is that alphabetical sorting is required so the comment would have to move from time to time, and this pollutes commits with unrelated changes.